presto-geospatial-toolkit/src/main/java/io/prestosql/geospatial/GeometryType.java

```diff
@@ -44,7 +44,7 @@ public boolean isMultitype()
 
     public Slice standardName()
     {
-        return standardName;
+        return utf8Slice(standardName);
```

Comment:
Slice is mutable, create new `utf8Slice` instance is safer, unless we have immutable slice.

Reply:
This will result in an extra conversion for every call to `ST_GeometryType()`, but maybe that's ok.

Reply:
I would do the UTF8 conversion once and then copy.  

Then again, Presto passes slice everywhere with the assumption that it will not be modified.  This was a purposeful trade off between performance and safety, so changing this in one place doesn't make sense to me.